docs/_includes/themes/zeppelin/_navigation.html

```diff
@@ -26,6 +26,8 @@
                 <li><a href="{{BASE_PATH}}/install/yarn_install.html">YARN Install</a></li>
                 <li><a href="{{BASE_PATH}}/install/virtual_machine.html">Virtual Machine Install</a></li>
                 <li role="separator" class="divider"></li>
+                <li><a href="{{BASE_PATH}}/install/upgrade.html">Upgrade</a></li>
```

Comment:
Thanks for the good docs @mwkang !
How about saying **Upgrade Zeppelin Version** or **Upgrade Version** instead of just **Upgrade** so that people can recognize easily ? : )